mappings/net/minecraft/world/dimension/Dimension.mapping

```diff
@@ -3,13 +3,22 @@ CLASS net/minecraft/class_1208 net/minecraft/world/dimension/Dimension
 	FIELD field_4785 world Lnet/minecraft/class_1150;
 	FIELD field_4786 generatorType Lnet/minecraft/class_1160;
 	FIELD field_4787 biomeSource Lnet/minecraft/class_1175;
+	FIELD field_4787 singletonBiomeSource Lnet/minecraft/class_1178;
```

Comment:
this issue again... the key got duplicated and the mappings build doesn't try to only use the good one...